source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputRenderer.mm

```diff
@@ -169,6 +169,9 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
             inputview = [[ACRInputLabelView alloc] initInputLabelView:rootView acoConfig:acoConfig adaptiveInputElement:inputBlck inputView:txtInput accessibilityItem:txtInput viewGroup:viewGroup dataSource:textInputHandler];
         }
         txtInput.delegate = textInputHandler;
+        if ([textInputHandler respondsToSelector:@selector(textFieldDidChange:)]) {
```

Comment:
verifying if we need a txtInput nullcheck

Reply:
I don't see any issue with this.